application/server/server.go

```diff
@@ -64,6 +65,11 @@ func NewConiksServer(conf *Config) *ConiksServer {
 		epochTimer: application.NewEpochTimer(conf.EpochDeadline),
 	}
 
+	// save the initial STR to be used for initializing auditors
+	initSTRPath := utils.ResolvePath(conf.InitSTRPath,
+		conf.ConfigFilePath)
```

Comment:
error: `conf.ConfigFilePath undefined`

Reply:
I would like to put a FIXME here: this saving should happen in `protocol/` level (i.e., when the server starts and updates)